src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -192,8 +194,15 @@
    (when advanced?
      [advanced-cartouche transaction modal?])])
 
-(defn- send-transaction-panel [{:keys [modal? transaction scroll advanced? symbol]}]
-  (let [{:keys [amount amount-text amount-error signing? to to-name sufficient-funds? in-progress? from-chat?]} transaction
+(defn update-amount-fn [timeout symbol decimals token]
+  (fn [amount]
+    (when @timeout
+      (utils/clear-timeout @timeout))
```

Comment:
Can you detail this part?

Reply:
Which one, timeout? Sure I can, though it wasn't introduced in this PR

Reply:
Oh OK didn't see it in GH diff view. If you can add a quick comment on the top of your head would be nice?

Reply:
yep

Reply:
Even better, I can remove the whole function. It's unused after the rebase